packages/factorywoman/tests/lib/db.js

```diff
@@ -0,0 +1,13 @@
+User = new Meteor.Collection('users');
+Animal = new Meteor.Collection('animals');
+
+'use strict';
+
+if (Meteor.isServer) {
```

Comment:
Because insertions are only made on the server side, too.
But as it is in lib and makes more sense...